index/store/moss/store.go

```diff
@@ -167,12 +168,21 @@ func New(mo store.MergeOperator, config map[string]interface{}) (
 		mo:      mo,
 		llstore: llStore,
 		llstats: llStats,
+		config:  config,
 	}
 	rv.s = &stats{s: &rv}
 	return &rv, nil
 }
 
 func (s *Store) Close() error {
+	if v, ok := s.config["mossAbortCloseEnabled"]; ok && v.(bool) == true {
```

Comment:
Probably need to do a second, additional if-check instead of doing the cast to bool on a single line.

The reason why is a failed cast can panic() the process / evil input can take down the server.  Quickie test of that here... https://play.golang.org/p/ErHjReFC72